### PR TITLE
vox mask pack 1

### PR DESCRIPTION
**Pull request by jack-thedragon95**
## About the PR
**MORE MASKS FOR VOX**

## Technical details
one more for civs and an exclusive sec mask :)

## Media
voxmask1

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added more vox masks SKREEEEEEE!!!!!

### CODE REVIEW

File: Resources/Prototypes/_Harmony/Loadouts/Miscellaneous/survival.yml
Comment (DieselMohawk) [inline Resources/Prototypes/_Harmony/Loadouts/Miscellaneous/survival.yml:1]: Currently this allows any vox to get a round start armored mask, are you able to make this Sec only somehow?

Comment (jack-thedragon95) [reply Resources/Prototypes/_Harmony/Loadouts/Miscellaneous/survival.yml:1]: its only in the sec loadout screen and has the sec restricted gear component

**Review by DieselMohawk — APPROVED** [review]
I see now, looks fine to me.

**Comment by FluffMe** [comment]
* Fixed `ClothingMaskNTEnforcer` to not have `HamsterWearable` tag.
* Removed redundant empty `Tag` declarations from `ClothingMaskVoxSpecial` and `ClothingMaskAuralis`
* Changed base inheritance of `ClothingMaskNTEnforcer` from `ClothingMaskGas` to `ClothingMaskGasSecurity` to inherit armor values and have same balance as security gas mask always.
* Removed redundant components from `ClothingMaskNTEnforcer` that are already declared in `ClothingMaskGas` and `ClothingMaskGasSecurity` and inherited: `Item`, `IdentityBlocker`, `BreathMask` and `Armor`

**Review by FluffMe — APPROVED** [review]

**Review by DieselMohawk — APPROVED** [review]
Looks good to me

**Pull request merged**